Cargo.lock

```diff
@@ -1653,6 +1653,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "7ced92e76e966ca2fd84c8f7aa01a4aea65b0eb6648d72f7c8f3e2764a67fece"
 dependencies = [
  "crc32fast",
+ "libz-rs-sys",
```

Comment:
Interesting how this got added here without a version bump... :thinking: :shrug:

Reply:
I'll admit I don't fully understand the meaning of the description here, but it seems related to this: https://github.com/zip-rs/zip2/pull/352#issue-3058459874